Makefile

```diff
@@ -35,7 +35,7 @@ help:
 
 .PHONY: generate-version-file
 generate-version-file: ## Generates the app version file
-	@echo -e "__travis_commit__ = \"${GIT_COMMIT}\"\n__time__ = \"${DATE}\"\n__travis_job_number__ = \"${BUILD_NUMBER}\"\n__travis_job_url__ = \"${BUILD_URL}\"" > ${APP_VERSION_FILE}
+	@echo -e "__git_commit__ = \"${GIT_COMMIT}\"\n__time__ = \"${DATE}\"\n__job_url__ = \"${BUILD_URL}\"" > ${APP_VERSION_FILE}
```

Comment:
I think that you can just remove `__job_url__` completely from this command and all references to `BUILD_URL` from the Makefile too. It looks like we stopped using that in notifications-aws a while ago.

Reply:
Ahh, good point, I did this now 👍